docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -105,6 +105,12 @@ which should produce output like this:
 
 The IP address is listed next to `LoadBalancer Ingress`.
 
+**Note**: If you are running your service on Minikube, you can find the assigned IP address and port with:
```

Comment:
Please add `{: .note}` underneath this line to make it a Note callout per https://kubernetes.io/docs/home/contribute/style-guide/#note?

Reply:
@heckj Done!